eth/tracers/native/prestate.go

```diff
@@ -47,10 +47,13 @@ type account struct {
 	Nonce   uint64                      `json:"nonce,omitempty"`
 	Storage map[common.Hash]common.Hash `json:"storage,omitempty"`
 	empty   bool
+
+	ArbitrumStorage  map[common.Hash]common.Hash `json:"arbitrumStorage,omitempty"`
+	arbStorageKeyMap map[common.Hash]common.Hash
```

Comment:
Prestate is just about finding all the places in storage that chaged.
We're not trying to add new things there. It already supports writes that happen during EVM by mimicing an SLOAD/SSTORE.
We just want to add the accesses that happened before EVM execution to the same place.
So we shouldn't need new entries. It should all be normal entries in the arbosstorage account.